src/libguac/Makefile.am

```diff
@@ -135,7 +135,7 @@ libguac_la_CFLAGS = \
     -Werror -Wall -pedantic
 
 libguac_la_LDFLAGS =     \
-    -version-info 18:0:1 \
+    -version-info 19:0:2 \
```

Comment:
I believe `age` needs to be reset to 0 as interfaces were changed (a new `protocol_version` member was added to the non-opaque `guac_user_info` structure). Code expecting the newer version of `guac_user_info` may not work (segfault) if it is given an older version and it attempts to access `protocol_version`.

If `guac_user_info` were opaque (with its members exposed only through getter functions), this would be correct.

Reply:
Okay, I have reset it to 0.